docs/research-software/vasp/vasp.md

```diff
@@ -188,6 +188,26 @@ versions in the build instructions GitHub repository. See:
 
 ## Tips for using VASP on ARCHER2
 
+### Switching MPI transport protocol rom UCX to OpenFabrics
```

Comment:
"from" is misspelled. The rest looks good to me.

Reply:
Well spotted, fixed now